### DISCUSSION TITLE
Thanks for losing my files!

### DISCUSSION THREAD

**Discussion by its-all-waves** [community/community#48574] [Codespaces] [answered]
### Select Topic Area

Show & Tell

### Body

Today, it's as if my codespace never existed! Found the backing repo to be MISSING AT LEAST A WEEK'S WORTH OF WORK. And the status page says "All systems are go!" or whatever BS is up there. 👍🏽

**Comment by dmilburn** [comment, marked as answer]
hi from the codespaces team,

over the weekend, the cs50 organization changed their GitHub plan by mistake which created some cascading changes, such as some codespaces getting deleted. we're sorry for the inconvenience. 

we've been able to restore almost all of the codespaces that were deleted. some students had already pushed all of their changes from their codespace to github. in that case, we won't be restoring the codespace directly since a new one can be created without any code loss.

If you continue to see any files missing, please go to https://code.cs50.io/ which has instructions you can follow to find any missing work


![Screenshot 2023-02-27 at 5 19 46 PM](https://user-images.githubusercontent.com/10053402/221698949-2ab3ad6f-2f87-441f-987f-62262a27460a.png)

**Reply by its-all-waves** [reply]
Yes, but why was I missing over a week of commits in the repo that *was not* deleted? How is that even possible? And how is it that those files were unrecoverable by the instructions posted at the last link you provided?

**Comment by Jeffrosenfeld** [comment]
I lost a lot of files as well and don't see any form at cs50.ly/files  just a 404 error.  Is there any other way to recover what was lost?

**Comment by its-all-waves** [comment]
What I don't get is how over a weeks worth of auto-commits were missing. To be clear, students don't commit--it's disabled. VScode is auto-committing every few minutes! There were literally hundreds of my commits missing.

Can GitHub please explain that? 

When a client changes something in their account--you all permanently delete commits? Doesn't add up.

And no, I never found a way to recover files. Filled out the form. Never saw my files again.

**Reply by its-all-waves** [reply]
@dmilburn
